### DISCUSSION TITLE
no_prefix strategy

### DISCUSSION THREAD

**Discussion by owljacob** [s00d/nuxt-i18n-micro#55] [Q&A] [answered]
Does such functionality exist? If not, are there any plans to add it?

**Comment by s00d** [comment, marked as answer]
no

I’ve already responded somewhere that I don’t see any point in this, you can simply integrate Vue i18n into the project. The instructions are here: https://vue-i18n.intlify.dev/guide/integrations/nuxt3.html
I'm not a big fan of no_prefix. SEO won't work, and a lot of new code and conditions will appear in the project, resulting in a second sluggish Nuxt i18n. Unfortunately, this is just a personal project that already consumes a lot of my personal time and effort, and I'm not ready to spend even more time on it.
